src/main/java/dog/pawbook/model/dog/Age.java

```diff
@@ -0,0 +1,51 @@
+package dog.pawbook.model.dog;
+
+import static dog.pawbook.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Dog's breed in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBreed(String)}
+ */
+public class Age {
+
+    public static final String MESSAGE_CONSTRAINTS = "Age should only contain numbers and should not be blank.";
+    public static final String VALIDATION_REGEX = "\\d+";
+
+    public final String value;
```

Comment:
I was wondering if it is better to store the date of birth and calculate the age from there.